core/domain/wipeout_service.py

```diff
@@ -1418,3 +1419,66 @@ def _pseudonymize_models_transactional(
             _pseudonymize_models_transactional(
                 [m for m in transaction_slice if m is not None],
                 pseudonymized_id)
+
+
+def _pseudonymize_version_history_models(pending_deletion_request):
+    """Pseudonymizes the version history models for the user with user_id.
+
+    Args:
+        pending_deletion_request: PendingDeletionRequest. The pending
+            deletion request object to be saved in the datastore.
+    """
+    user_id = pending_deletion_request.user_id
+
+    version_history_model_class = exp_models.ExplorationVersionHistoryModel
+    version_history_models = version_history_model_class.query(
+        user_id == version_history_model_class.committer_ids
+    ).fetch()
+
+    save_pending_deletion_requests([pending_deletion_request])
```

Comment:
Why do we do this?

Reply:
Owh I forgot to remove this when I removed the newly created pseudonymous ID. Removed it now.